### ISSUE TITLE
Material Grader won't import/export items.

### ISSUE THREAD

**Issue by thaumictom** [SilentChaos512/SilentGems#236]
### Versions
Minecraft Version: 1.12.2 
Modpack (if avaible): FTB Horizons III
Modpack Version (if avaible): 1.1.0
Silent's Gems Version: 2.5.8
Silent Lib Version:  2.2.11

### Expected behavior
Material Grader should export items with pipes/hopper from the bottom side.

### Actual behaviour
Nothing happens.

### Steps to reproduce the problem
Place a Material Grader with a Hopper under it, nothing will happen.

### Special Info
I put End Diamonds (from Calculator Mod) as they work as Super Gems in, so I am not sure if it works with other Gems.

**Comment by SilentChaos512** [comment]
No issue with just Gems and Calculator installed. Something in Horizons must be interfering with hoppers.

**Comment by silvanasmax** [comment]
Confirmed bug.  On Horizons 1.0.0 (Server #1), I used the Material Grader with hoppers, and they exported fine.  As of Horizons 1.1.0, I moved to Server #2, and set up the same process, and it failed to go through hoppers, even after verifying that chunkloading, land claiming, etc for that server was accounted for.  Moving back to Server #1 under the same version, I verified it was broken.

Further testing using Thermal item pipes w/ & w/o servos also failed to dump items, as well as connecting to different sides.  Importing through the top using various means still works, however.
